API.md

```diff
@@ -315,6 +315,13 @@ Open MCT is packaged along with a few general-purpose plugins:
   user to store created items.
 * `openmct.plugins.utcTimeSystem` provides support for using the time
   conductor with UTC time.
+* `openmct.plugins.URLIndicatorPlugin({
+    url: 'http://google.com',
+    icon: 'check',
+    interval: '10E3'
+  })` adds an indicator which shows the
+  current status of a given URL with the three labels `url` (mandatory), 
+  `icon` (defaults to `database`) and `interval` (defaults to `1000`). 
```

Comment:
Thought I had that right, 10E3 is 10*10*10 = 1000, I'll recheck. Do you want it to be anything else than 1sec?